conf/Config.js

```diff
@@ -53,6 +53,12 @@ class Config {
             this.healthChecks = { allowFrom: ['127.0.0.1/8', '::1'] };
         }
 
+        // TODO: Drop this before merge
```

Comment:
Something to do here? :)

Reply:
haha yup, this needs to be dropped before merging as it is a placeholder for redis configs (defined in metrics accumulator PR#125)